### ISSUE TITLE
DaCosta-20?-2016

### ISSUE THREAD

**Issue by LinguList** [clld/concepticon-data#152]
http://www.lrec-conf.org/proceedings/lrec2016/workshops/LREC2016Workshop-GLOBALEX_Proceedings.pdf#page=35

This is interesting, since they do similar stuff to ours, but they link directly to wordnet and it's not clear whether they follow things like our guidelines (I suspect it's more towards automatic stuff).

**Comment by xrotwang** [comment]
It looks like they are doing almost exactly the same :) although we seem to be ahead of them (except the OMW thing, which I don't know much about). Maybe we should just email them, pointing to concepticon.clld.org and ask what they think about it?

**Comment by LinguList** [comment]
Yep, I already did it. I'll keep you posted and cc you if they answer.
